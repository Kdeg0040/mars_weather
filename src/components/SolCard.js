import React, {Component} from 'react';

class SolCard extends Component {
  render () {
    return <div></div>
  }
}

module.exports = SolCard;