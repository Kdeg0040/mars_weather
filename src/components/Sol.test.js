import React from 'react';
import { shallow } from 'enzyme';

import SolCard from './SolCard';

const solCard = shallow(<SolCard />);

describe('<SolCard />', () => {
  it('renders correctly', () => {
    expect(solCard).toMatchSnapshot();
  });
});