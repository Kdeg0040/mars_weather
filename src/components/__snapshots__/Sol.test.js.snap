// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SolCard /> renders correctly 1`] = `ShallowWrapper {}`;
